src/challenge/Challenge.tsx

```diff
@@ -356,11 +386,20 @@ class Challenge extends React.Component<ChallengeProps, ChallengeState> {
                   visible={this.getVisibilityWithHack(
                     !this.state.editorFullScreen
                   )}
-                  maxSize={350}
+                  maxSize={150}
                   minSize={150}
                 >
                   {this.renderConsole()}
                 </Allotment.Pane>
+                <Allotment.Pane
+                  visible={this.getVisibilityWithHack(
+                    !this.state.editorFullScreen
+                  )}
+                  maxSize={400}
+                  minSize={400}
+                >
+                  {this.renderGraphicsPane()}
+                </Allotment.Pane>                
```

Comment:
OK for proof of concept, but I wonder if maybe we should have a challenge typ(e) which is `graphics`, andonly show this for that type.
Also, could consider showing graphics in place of the console or tabbed.